understanding/22/target-size-minimum.html

```diff
@@ -23,13 +23,13 @@ <h4>Target Size (Minimum)</h4>
              <p class="change">New</p>
 
              <p>The size of the <a>target</a> for <a>pointer inputs</a> is at least 24 by 24 <a>CSS pixels</a>, except where:</p>
-            <ul>
-                <li><strong>Spacing:</strong> The target does not overlap any other target and has a <a>target offset</a> of at least 24 CSS pixels to every adjacent target;</li>
+             <ul>
+                <li><strong>Spacing:</strong> Undersized targets (those less than 24 by 24 CSS pixels) are positioned so that if a 24 CSS pixel diameter circle is centered on the <a>bounding box</a> of each, the circles do not intersect another target or the circle for another undersized target;</li> 
```

Comment:
does this need a new definition somewhere for "bounding box"?

Reply:
probably something very high-level like https://developer.mozilla.org/en-US/docs/Glossary/Bounding_box (rather than that definition of "minimum bounding box" that used to be used for focus appearance)

Reply:
Well, seeing as focus-appearance doesn't use that definition any more, I think we can re-use it here. The latest push includes that change.

Reply:
I do think the definition on https://developer.mozilla.org/en-US/docs/Glossary/Bounding_box is quite nice and concise. also, not quite sure what "For components which wrap onto multiple lines as part of a sentence or block of text (such as hypertext links), the bounding box is based on how the component would appear on a single line." part of the minimum bounding box definition actually means/shakes out to, so if we ARE going with that definition, this may need some clarification

Reply:
I agree that the Mozilla bounding box is definition seems sufficient.  We have the wrinkle that _our_ bounding box for hypertext is to based on the bounding box formed when linked text is unspooled to a single line.